README_zh_CN.md

```diff
@@ -89,39 +95,68 @@ make
 ## 添加一个云存储服务
 如果你打算仅在POSIX兼容文件系统中使用`mc`,那你可以直接略过本节，跳到[日常使用](#everyday-use)。
 
-添加一个或多个S3兼容的服务，请参考下面说明。`mc`将所有的配置信息都存储在``~/.mc/config.json``文件中。
+添加一个或多个S3亚马逊兼容的服务，请参考下面说明。`mc`将所有的配置信息都存储在``~/.mc/config.json``文件中。
 
 ```sh
-mc config host add <ALIAS> <YOUR-S3-ENDPOINT> <YOUR-ACCESS-KEY> <YOUR-SECRET-KEY> <API-SIGNATURE>
+mc config host add <ALIAS> <YOUR-S3-ENDPOINT> <YOUR-ACCESS-KEY> <YOUR-SECRET-KEY> --api <API-SIGNATURE> --lookup <BUCKET-LOOKUP-TYPE>
 ```
 
-别名就是给你的云存储服务起了一个短点的外号。S3 endpoint,access key和secret key是你的云存储服务提供的。API签名是可选参数，默认情况下，它被设置为"S3v4"。
+别名只是你云存储服务的简称。VS3终端（endpoint），访问密钥（access key）和密钥（secret key）是由你的云存储提供商提供的。API签名是可选参数，默认情况下，它被设置为"S3v4"。
+
+Lookup是一个可选参数。它被用于指示dns和path样式的url请求是否被服务器支持，它接受“dns”，“path”和"auto"作为有效值。在默认情况下，它被设为“auto”，而且SDK会自动地决定所使用的url查找类型。
 
-### 示例-MinIO云存储
-从MinIO服务获得URL、access key和secret key。
+### 示例 - MinIO云存储
+从MinIO服务获得URL、访问密钥（access key）和密钥（secret key）。
 
 ```sh
-mc config host add minio http://192.168.1.51 BKIKJAA5BMMU2RHO6IBB V7f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12 S3v4
+mc config host add minio http://192.168.1.51 BKIKJAA5BMMU2RHO6IBB V7f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12
 ```
 
-### 示例-Amazon S3云存储
+### 示例 - Amazon S3云存储
 参考[AWS Credentials指南](http://docs.aws.amazon.com/general/latest/gr/aws-security-credentials.html)获取你的AccessKeyID和SecretAccessKey。
 
 ```sh
-mc config host add s3 https://s3.amazonaws.com BKIKJAA5BMMU2RHO6IBB V7f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12 S3v4
-```
-
-### 示例-Google云存储
+mc config host add s3 https://s3.amazonaws.com BKIKJAA5BMMU2RHO6IBB V7f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12 
+```
+
+**注意**：作为Amazon S3上的一个IAM使用者，你需要确保用户对存储桶的完全访问或为你的IAM用户设置下列限定的策略
+
+```json
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Sid": "AllowBucketStat",
+            "Effect": "Allow",
+            "Action": [
+                "s3:HeadBucket"
+            ],
+            "Resource": "*"
+        },
+        {
+            "Sid": "AllowThisBucketOnly",
+            "Effect": "Allow",
+            "Action": "s3:*",
+            "Resource": [
+                "arn:aws:s3:::<your-restricted-bucket>/*",
+                "arn:aws:s3:::<your-restricted-bucket>"
+            ]
+        }
+    ]
+}
+```
+
+### 示例 - Google云存储
 参考[Google Credentials Guide](https://cloud.google.com/storage/docs/migrating?hl=en#keys)获取你的AccessKeyID和SecretAccessKey。
 
 ```sh
-mc config host add gcs  https://storage.googleapis.com BKIKJAA5BMMU2RHO6IBB V8f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12 S3v2
+mc config host add gcs  https://storage.googleapis.com BKIKJAA5BMMU2RHO6IBB V8f1CwQqAcwo80UEIJEjc5gVQUSSx5ohQ9GSrr12 
 ```
 
 注意：Google云存储只支持旧版签名版本V2，所以你需要选择S3v2。
 
-## 验证
-`mc`预先配置了云存储服务URL：https://play.min.io:9000，别名“play”。它是一个用于研发和测试的MinIO服务。如果想测试Amazon S3,你可以将“play”替换为“s3”。
+## 验证你的设置
+`mc`预先配置了云存储服务URL：https://play.minio.io:9000，别名“play”。它是一个用于研发和测试的MinIO服务。如果想测试Amazon S3,你可以用“s3”或者你在开始设置的别名来替换“play”。
```

Comment:
This ends up being rendered as a link to the browser, if we mean for that to happen then it should link to play.minio.io:9000, but as it is it thinks the full link is
```
https://play.minio.io:9000，别名“play”。它是一个用于研发和测试的MinIO服务。如果想测试Amazon
```
In chrome at least this link ends up being blocked.

Reply:
Thanks for your suggestion. I'll correct it.